src/REPLMode/REPLMode.jl

```diff
@@ -717,13 +717,17 @@ function try_prompt_pkg_add(pkgs::Vector{Symbol})
         API.add(string.(available_pkgs))
     elseif lower_resp in ["o"]
         editable_envs = filter(v -> v != "@stdlib", LOAD_PATH)
-        expanded_envs = Base.load_path_expand.(editable_envs)
-        envs = convert(Vector{String}, filter(x -> !isnothing(x), expanded_envs))
         option_list = String[]
         keybindings = Char[]
-        for i in 1:length(envs)
-            push!(option_list, "$(i): $(pathrepr(envs[i])) ($(editable_envs[i]))")
-            push!(keybindings, only("$i"))
+        for i in 1:length(editable_envs)
```

Comment:
I think it's reasonable to limit this to 9
```suggestion
        for i in 1:min(length(editable_envs), 9)
```